docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,135 @@
+=====================================
+Writing your first Django app, part 8
+=====================================
+
+This tutorial begins where :doc:`Tutorial 7 </intro/tutorial07>` left off. We've
+built our web-poll application and will now look at third-party packages. One of
+Django's strengths is the rich ecosystem of third-party packages. They're
+community developed packages that can be used to quickly improve the feature-set
+of an application.
+
+This tutorial will show how to add the `Django Debug Toolbar
+<https://django-debug-toolbar.readthedocs.io>`_, a commonly used third-party
+package. From 2020-2022, the Django Debug Toolbar has ranked in the top three
+most used third-party packages according to the Django Developer Survey (
+`2022 <https://lp.jetbrains.com/to-be-released-yet>`_,
+`2021 <https://lp.jetbrains.com/django-developer-survey-2021-486/#horizontal-bar-chart-109>`_,
+`2020 <https://docs.google.com/forms/d/15GRScagj29AjfFh1Js0ujCEQ2EUqMer5nMB3Fp1KoG8/viewanalytics#21>`_,
+).
+
+
+.. admonition:: Where to get help:
+
+    If you're having trouble going through this tutorial, please head over to
+    the :doc:`Getting Help</faq/help>` section of the FAQ.
+
+Installing the Django Debug Toolbar
+===================================
+
+A useful tool for debugging Django web applications is the Django Debug
+Toolbar. It's a third-party package maintained by the `Jazzband
+<https://jazzband.co>`_ organization. The toolbar helps you understand how your
+application functions and to identify problems. It does so by providing panels
+that provide information about a specific system of Django applications.
```

Comment:
I found this tricky to follow. The DDT readme has a nice description. Trying to use that here maybe something like....

```suggestion
that provide debug information about the current request/response.
```